cmd/kogito/command/shared/converters.go

```diff
@@ -16,17 +16,42 @@ package shared
 
 import (
 	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/util"
```

Comment:
Question: you removed from the util package because these functions were only used in here? Many thanks for it!

Reply:
Previous `FromStringsKeyPairToMap()` is written `map.go` file under kogito-cloud-operator package but it is only used in kogito-cli -> `converters.go` file, so I moved this function from util package to converter file.